src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -81,4 +81,33 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+    /** Stores the User in Datastore. */
+    public void storeUser(User user) {
+        Entity userEntity = new Entity("User", user.getEmail());
+        userEntity.setProperty("email", user.getEmail());
+        userEntity.setProperty("aboutMe", user.getAboutMe());
+        datastore.put(userEntity);
+    }
+
+    /**
+     * Returns the User owned by the email address, or
+     * null if no matching User was found.
+     */
+    public User getUser(String email) {
+
+        Query query = new Query("User")
+                .setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
+        PreparedQuery results = datastore.prepare(query);
+        Entity userEntity = results.asSingleEntity();
+        if(userEntity == null) {
+            return null;
+        }
+
+        String aboutMe = (String) userEntity.getProperty("aboutMe");
```

Comment:
What happens here if the property hasn't been set? Does it throw? Does it produce a null value? Should there be any sort of logic to default the value to "" when the property is unset/null?